src/config.js

```diff
@@ -65,6 +65,7 @@ export const urls = {
   thirdPartyFrameHost: env['thirdPartyFrameHost'] || 'ampproject.net',
   thirdPartyFrameRegex,
   cdn:
+    // TODO(danielrozenberg): replace default hostname with ampjs.org.
     env['cdnUrl'] || getMetaUrl('runtime-host') || 'https://cdn.ampproject.org',
```

Comment:
I think we should be creating a second variable, one for JS URLs and one for Cache URLs.

Reply:
I can do search & replace pretty easily but more than that requires more familiarity with the runtime than I have 😅